packages/jest-preset-default/scripts/pegjs-transform.js

```diff
@@ -1,16 +0,0 @@
-const pegjs = require( 'pegjs' );
```

Comment:
The same applies - `pegjs` transform is duplicated here and we need it only for Gutenberg.